### PR TITLE
updates the urbit formula to download a binary dependency

### PR DESCRIPTION
**Pull request by joemfb**
The `urbit.pill` binary dependency is no longer checked-in to source control, which broke this formula.

**Question**: is there a mechanism for renaming the contents of a resource? (maybe via the constructor, or `resource.stage`) That would make the post_install step unnecessary.

re urbit/urbit#721

### CODE REVIEW

**Comment by galenwp** [comment]
Hrm.  Looks like the build failed, but [the stacktrace](http://bot.brew.sh/job/Homebrew%20Head-Only%20Pull%20Requests/371/version=el_capitan/testReport/brew-test-bot/el_capitan/audit_homebrew_head_only_urbit/) isn't all that meaningful to me.  No need for `:using`?

**Comment by joemfb** [comment]
Yeah, the `:using` param was redundant, and the usage message had changed, failing the test. I've fixed both of those. I don't think I can make `brew audit` pass, though, as this is necessary "Head-only".

**Comment by joemfb** [comment]
And I just noticed that this is a duplicate of #204

**Comment by galenwp** [comment]
Huh!  Not sure how I missed #204.  Sorry @Spaceghost.  If this build passes hopefully someone from the Homebrew team can review.

**Comment by joemfb** [comment]
Yup, everything passed but `brew audit`, which is failing the build on the basis of it being ["Head-only"](http://bot.brew.sh/job/Homebrew%20Head-Only%20Pull%20Requests/372/version=mavericks/testReport/brew-test-bot/mavericks/audit_homebrew_head_only_urbit/) ... 

This formula is being detected as a ["first time build"](http://bot.brew.sh/job/Homebrew%20Head-Only%20Pull%20Requests/372/version=mavericks/console), which I think is triggering the inclusion of `brew audit` (per #186).

**Comment by Spaceghost** [comment]
This is definitely better than #204. No worries on that one going into the ether, because #205 is superior. Thank you @joemfb!

File: urbit.rb [hunk 1]
```diff
@@ -12,18 +12,25 @@ class Urbit < Formula
   depends_on "automake" => :build
   depends_on "cmake" => :build
 
+  resource "latest.pill" do
+    url "http://bootstrap.urbit.org/latest.pill"
```
Comment (bfontaine) [inline urbit.rb]: Please add a checksum for this resource. It should look like this:
```rb
resource "..." do
  url "..."
  sha256 "..."
end
```

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: urbit.rb [hunk 2]
```diff
@@ -12,18 +12,25 @@ class Urbit < Formula
...
+
   def install
     system "make", "BIN=#{bin}", "LIB=#{share}"
-    share.install "urb/urbit.pill"
-    share.install Dir["urb/zod/*"]
+    resource("latest.pill").stage share
+  end
+
+  def post_install
+    mv share/"latest.pill", share/"urbit.pill"
```
Comment (bfontaine) [inline urbit.rb]: Why not do that during install?

Comment (joemfb) [inline urbit.rb]: I think everything pill-related in this formula is obsolete -- the latest `urbit` binary automatically downloads the pill: https://github.com/urbit/urbit/commit/bbc61d841bf5855db9a5d2cf4ba83f00db49249c.

Also, I think there's ongoing work to version releases; @juped, is that correct? Do you want to fix this head-only installer, or will you be ready for a "normal" formula soon?

**Pull request closed without merging**